### ISSUE TITLE
[plugin:vite:css] Preprocessor dependency "sass" not found. Did you install it?

### ISSUE THREAD

**Issue by productfrontenddeveloper** [nuxt/vite#23]
```
[plugin:vite:css] Preprocessor dependency "sass" not found. Did you install it?
/my-proje/assets/css/tailwind.scss
    at loadPreprocessor (/my-proje/node_modules/vite/dist/node/chunks/dep-00e79b84.js:18188:15)
    at scss (/my-proje/node_modules/vite/dist/node/chunks/dep-00e79b84.js:18193:20)
    at compileCSS (/my-proje/node_modules/vite/dist/node/chunks/dep-00e79b84.js:17965:40)
    at TransformContext.transform (/my-proje/node_modules/vite/dist/node/chunks/dep-00e79b84.js:17653:50)
    at Object.transform (/my-proje/node_modules/vite/dist/node/chunks/dep-00e79b84.js:45957:30)
    at transformRequest (/my-proje/node_modules/vite/dist/node/chunks/dep-00e79b84.js:61528:29)
    at /my-proje/node_modules/vite/dist/node/chunks/dep-00e79b84.js:61630:32
Click outside or fix the code to dismiss.
You can also disable this overlay with hmr: { overlay: false } in vite.config.js.
```

**Comment by cretueusebiu** [comment]
`yarn add --dev sass` / `npm i -D sass`

**Comment by diasfs** [comment]
yarn add --dev sass don't solves it

**Comment by productfrontenddeveloper** [comment]
yes some problem reload

**Comment by phamhongphuc** [comment]
rollback to version `0.1.0` solved the problem

**Comment by euoia** [comment]
> rollback to version nuxt-vite@0.1.0 solved the problem

Just chiming in to say I had the same problem and it was solved by rolling back to `nuxt-vite@0.1.0`.

**Comment by productfrontenddeveloper** [comment]
new version fixed, yarn lock delete and nuxt vite 0.2.3 fixed
